tests/core/test_db.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import pytest_asyncio
+
+from lnbits.db import QueryValues
+
+
+@pytest_asyncio.fixture(autouse=True)
+async def json_test_table(db):
+    try:
+        await db.execute("CREATE TABLE test (data JSON)")
+        yield db
+    finally:
+        await db.execute("DROP TABLE test")
+
+
+@pytest.mark.asyncio
+async def test_sql_json(db):
+    obj = {"a": 3, "b": "bar", "c": {"nested": "d"}}
+    another_obj = {"a": 2, "b": "baz"}
+
+    await db.execute("INSERT INTO test VALUES(?)", (obj,))
+    await db.execute("INSERT INTO test VALUES(?)", (another_obj,))
+
+    values = QueryValues()
+    rows = await db.fetchall(
+        f"""
+        SELECT * FROM test WHERE {values.json_path('data', 'a', type_=int)} >= {values(obj["a"])}
+        """,
+        values,
+    )
+    assert len(rows) == 1
+    assert rows[0].data == obj
+
+    values = QueryValues()
+    rows = await db.fetchall(
```

Comment:
- there is the risk that the parameters for `json_path` will be built dynamically
- there is no check for these values, below I have managed to insert a new row (SQL Injection)
- we should not rely on the fact that developers will use the API correctly (use a hardcoded value for the "column" param)
```py
    dataColumn = """data IS NOT NULL; INSERT INTO test VALUES ('{"inject": 3}'); SELECT * FROM test WHERE data"""
    rows = await db.fetchall(
        f"SELECT * FROM test WHERE {values.json_path(dataColumn, 'c', 'nested')} = {values(obj['c']['nested'])}",
        values,
    )
```

Reply:
The column name unfortunately cant be passed as a value, and usually the dynamic part would be the path and not the column

Reply:
https://www.psycopg.org/docs/extensions.html#psycopg2.extensions.quote_ident exists but it seems like a pg-only thing and also requires a connection

Reply:
- the column name could be sanitized as described in the first part of this: https://www.postgresql.org/docs/7.0/syntax525.htm 
> Names in SQL must begin with a letter (a-z) or underscore (_). Subsequent characters in a name can be letters, digits (0-9), or underscores. 

- I do not think it is necessary to cover the special cases where names are surrounded in double quotes (second part of the doc)

Reply:
i added a somewhat primitive `_sanitize` function. if there is any non-alphabetic character, it rejects. Do you think this is sufficient?